recipe/meta.yaml

```diff
@@ -1,56 +1,70 @@
-{% set name = "Flask-Cors" %}
-{% set version = "3.0.10" %}
-{% set bundle = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash = "b60839393f3b84a0f3746f6cdca56c1ad7426aa738b70d6c61375857823181de" %}
+{% set name = "flask-cors" %}
+{% set version = "6.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  url: https://github.com/corydolphin/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 229f273a5dd5ba6095ff162771d182fdc26070efa074fb83d54d4e83d0974405
+  patches:
+    - patches/0001-fix-hardcoded-version.patch
```

Comment:
Can you add a comment that we should update the version in the patch **manually** until the maintainers solve the issue https://github.com/corydolphin/flask-cors/issues/387?